src/frontend/src/components/SEO.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import useSiteMetadata from '../hooks/use-site-metadata';
+
+function SEO({ description_, lang, meta, title_ }) {
+  const { title, description } = useSiteMetadata();
+  const metaDescription = description_ || description;
+
+  return (
+    <Helmet
+      htmlAttributes={{
+        lang,
+      }}
+      title={title_}
```

Comment:
I'm getting `<title></title>` on Zeit.  I think we need to make sure that one of `title` or `title_` are set, like you do with description above.